doc/cli/npm-ls.md

```diff
@@ -3,10 +3,9 @@ npm-ls(1) -- List installed packages
 
 ## SYNOPSIS
 
-    npm list [[@<scope>/]<pkg> ...]
-    npm ls [[@<scope>/]<pkg> ...]
-    npm la [[@<scope>/]<pkg> ...]
-    npm ll [[@<scope>/]<pkg> ...]
+    npm ls [[<@scope>/]<pkg> ...]
+
+    aliases: list, la, ll
```

Comment:
It's worth noting at some point that `npm ll` and `npm la` do something very different from `npm ls` (try them!), but that doesn't need to be part of this commit.